### DISCUSSION TITLE
Elliptic Curves Cryptography implementation

### DISCUSSION THREAD

**Discussion by longer--** [krzyzanowskim/CryptoSwift#927] [General]
Elliptic Curves Cryptography is an exciting algorithm, it would be very great to be implemented in CryptoSwift.

**Comment by mrdotmo** [comment]
Is this still an open issue?  Mind if I stake stab?

**Comment by ChoshimWy** [comment]
Yes, I think it is very nice to add elliptic curve encryption algorithm
